packages/tpetra/core/src/Tpetra_MultiVector_decl.hpp

```diff
@@ -1466,14 +1466,32 @@ namespace Tpetra {
     /// This requires that there are no live host-space views.
     typename dual_view_type::t_dev::const_type getLocalViewDevice(Access::ReadOnlyStruct) const;
 
+    /// \brief Return a read-only, up-to-date view of this MultiVector's local data on device.
+    /// This requires that there are no live host-space views.
+    /// WARNING: This function will only synchronize the provided execution_space instance, which if not used correctly
+    /// can lead to errors.
```

Comment:
```suggestion
    /// @warning This function will only synchronize the provided @c execution_space instance, which if not used correctly can lead to errors.
```
(is more `Doxygen` friendly)